src/aggregator/runtime/options_manager.go

```diff
@@ -36,7 +36,7 @@ type OptionsManager interface {
 	// RegisterWatcher registers a watcher that watches updates to runtime options.
 	// When an update arrives, the manager will deliver the update to all registered
 	// watchers.
-	RegisterWatcher(l OptionsWatcher) close.SimpleCloser
+	RegisterWatcher(l OptionsWatcher) resource.Closer
```

Comment:
I know we're not doing anything about it right now, but we'll eventually want to either (a) promote `resource` to non-`x/` or (b) create per-package interfaces instead of bleeding `x/` types as part of non-`x/` APIs.